roles/update/templates/workload_launch.sh.j2

```diff
@@ -319,6 +324,9 @@ function workload_launch {
             --ram {{ workload_memory|default(512) }} \
             --disk {{ workload_disk|default(10) }} \
             --swap {{ workload_swap|default(1) }} \
+            --property hw:mem_page_size=large \
```

Comment:
For non-nfv workload I don't think we should set this properties.
I suggest to create nfv flavor and non-nfv inside if-else condition

Reply:
@ciecierski ack

Reply:
done